tutorials/01-Introduction.ipynb

```diff
@@ -0,0 +1,2872 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "bd2a49b7-fb04-4b00-a96e-cd1723711363",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 0.1$"
+      ],
+      "text/plain": [
+       "0.1000000000"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from sympy import *\n",
+    "\n",
+    "Float('0.1', 10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6583d34f-7e2e-43e4-ac79-408363487d97",
+   "metadata": {},
+   "source": [
+    "We note that the best way to create floats is to parse from strings.\n",
+    "\n",
+    "If you build float from python float literals, \n",
+    "the precision is limited to the python's floating points (17 decimal places), \n",
+    "and you may get unexpected result."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ca8cc0ae-8834-4646-ab6a-9299ff1f9477",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Float(0.1, 20) == Float('0.1', 20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e396a1e-634c-4628-9fbd-cdaff68015b4",
+   "metadata": {},
+   "source": [
+    "Getting decimal representation of rational numbers in any digits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "20ac9e71-13fc-4727-a955-10232e8125b4",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 0.1428571429$"
+      ],
+      "text/plain": [
+       "0.1428571429"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Float('1/7', 10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "decfeda2-de4f-4c93-8d20-af0b86b0fc9a",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$\\displaystyle 0.14285714285714285714$"
+      ],
+      "text/plain": [
+       "0.14285714285714285714"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Float('1/7', 20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0a6a676-58e1-4fca-bccf-910114c2cc4a",
+   "metadata": {},
+   "source": [
+    "The more advanced numeric evaluation methods will be explained in the next section."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c304abb-5e7f-4610-9348-e202eca71fa8",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Symbols"
```

Comment:
Let's make sure we discuss the gotchas around symbol names vs. variable names. Again, feel free to just copy text from the tutorial https://docs.sympy.org/latest/tutorials/intro-tutorial/gotchas.html#symbols